source/common/network/connection_impl.h

```diff
@@ -86,9 +86,9 @@ class ConnectionImpl : public virtual Connection,
     uint64_t bytes_processed_;
   };
 
-  virtual void closeSocket(uint32_t close_type);
+  virtual void closeSocket(ConnectionEvent close_type);
   void doConnect();
-  void raiseEvents(uint32_t events);
+  void raiseEvents(ConnectionEvent event);
```

Comment:
Could be renamed raiseEvent to match onEvent (which happened to be wrong before and is correct now :)